charts/prometheus-postgres-exporter/Chart.yaml

```
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: "0.10.1"
+appVersion: "0.11.1"
 description: A Helm chart for prometheus postgres-exporter
 name: prometheus-postgres-exporter
-version: 3.1.5
+version: 3.1.6
```

new featuer needs to raise minor version.

```suggestion
version: 3.2.0
```